### PR TITLE
Documentation for Diagrams

### PR DESCRIPTION
**Pull request by TravisFallsMSFT**
# Type of PR

- [ ] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Does this introduce a breaking change

- [ ] YES
- [ ] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes #issue_number (this will automatically close the issue when the PR closes)
- References #issue_number (this references the issue but does not close with PR)

### CODE REVIEW

File: docs/Automation-Observability.md [hunk 1]
```diff
@@ -67,12 +67,80 @@ Run the script to create the Diagnostic Setting.
...
+#### Dashboard Elements
+
+![Dashboard](./assets/Automation-Dashboard/DashboardForAutomation.png)
+Figure 2:  Grafana Automation Dashboard
+
+Once a user has logged in the dashboard, depicted in Figure 2 will display.  The graph has several areas of note that will each be discussed below.  The following list is a list and description of the elements.  
+
+| Element Name | Element Description | Figure Number | Image |
+| ------------ | ------------------- | ------------- | ----- |
+| Login | Allows for seamless login to the Grafana environment. | [Figure 1](#log-into-automation-dashboard) | [<img src="./assets/Automation-Dashboard/grafanaLogIn.png" width="250"/>](./assets/Automation-Dashboard/grafanaLogIn.png) |
```
Comment (AAkindele) [inline docs/Automation-Observability.md:90]: updated hash link
```suggestion
| Login | Allows for seamless login to the Grafana environment. | [Figure 1](#log-in-for-dashboard) | [<img src="./assets/Automation-Dashboard/grafanaLogIn.png" width="250"/>](./assets/Automation-Dashboard/grafanaLogIn.png) |
```

File: docs/Automation-Observability.md [hunk 2]
```diff
@@ -67,12 +67,80 @@ Run the script to create the Diagnostic Setting.
...
+Figure 3:  Grafana time line selection
+
+As show above a user can use the dropdown to adjust the timeline by which the data in the graph is loaded and thus a person can do analysis to ascertain the health of the automation process.  As shown in [Figure 3](#time-range-selector).
+
+#### Number of Jobs Created
+
+![Jobs Created](./assets/Automation-Dashboard/grafanaJobs-Created.png)
+Figure 4:  Grafana Jobs Created
+
+In [figure 4](#number-of-jobs-created) the number of jobs created to run during this time segment is listed.  This number is misleading, as it does not mean "new automation jobs" were created, but rather automation job instances created for running.  This is similar to the concept in American game baseball of "[On Deck](https://www.dictionary.com/browse/on--deck)".
```
Comment (AAkindele) [inline docs/Automation-Observability.md:113]: Should we change the title of this section of the dashboard? We could also include "automation job instances created for running" somewhere in the description tooltip in the dashboard.

**Review by AAkindele — APPROVED** [review]
🚀

File: docs/Automation-Observability.md (hunk 2 above)
Comment (TravisFallsMSFT) [reply docs/Automation-Observability.md:113]: I like this suggestion, changing now.

**Pull request merged**